README.md

```diff
@@ -253,7 +232,7 @@ If you like this project...
 
 > Give credit to everyone who inspired your codebase.
 
-I would like to thank...
+I would like to thank Queenter Juma you can follow her on github. Click this link >> (<a href="#readme-top">https://github.com/QueenterJuma</a>)
```

Comment:
- [ ] While you did a good job adding the required information on your README :100: I think it would be better if you give the appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).